docs/resources/deployment-examples/example-fargate/terraform/codebuild_config.tf

```diff
@@ -0,0 +1,269 @@
+
+locals {
+  subnet_arns = formatlist(
+    "arn:aws:ec2:us-east-2:245943599471:subnet/%s",
+    module.vpc.private_subnets
+  )
+}
+
+resource "aws_ecr_repository" "anchor_config" {
+  name                 = "${var.environment}-anchor-config"
+  image_tag_mutability = "MUTABLE"
+
+  image_scanning_configuration {
+    scan_on_push = true
+  }
+}
+
+resource "aws_s3_bucket" "anchor_config" {
+  bucket = "${var.environment}-anchor-config"
+  tags = {
+    Environment = "${var.environment}"
+  }
+}
+
+resource "aws_s3_bucket_acl" "anchor_config" {
+  bucket = aws_s3_bucket.anchor_config.id
+  acl    = "private"
+}
+
+resource "aws_s3_bucket_object" "anchor_config" {
+  bucket = aws_s3_bucket.anchor_config.id
+  key    = "anchor_config.yaml"
+  acl    = "private" # or can be "public-read"
+  source = "../example_config/anchor_config.yaml"
+  etag   = filemd5("../example_config/anchor_config.yaml")
+}
+resource "aws_s3_bucket_object" "reference_config" {
+  bucket = aws_s3_bucket.anchor_config.id
+  key    = "reference_config.yaml"
+  acl    = "private" # or can be "public-read"
+  source = "../example_config/reference_config.yaml"
+  etag   = filemd5("../example_config/reference_config.yaml")
+}
+resource "aws_s3_bucket_object" "stellar_toml" {
+  bucket = aws_s3_bucket.anchor_config.id
+  key    = "stellar.toml"
+  acl    = "private" # or can be "public-read"
+  source = "../example_config/stellar.toml"
+  etag   = filemd5("../example_config/stellar.toml")
+}
+
+resource "aws_iam_role" "codebuild_role" {
+  name = "${var.environment}-anchorplatform-codebuild"
+
+  assume_role_policy = <<EOF
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Effect": "Allow",
+      "Principal": {
+        "Service": "codebuild.amazonaws.com"
+      },
+      "Action": "sts:AssumeRole"
+    }
+  ]
+}
+EOF
+}
+
+resource "aws_iam_role_policy" "codebuild_policy" {
+  role = aws_iam_role.codebuild_role.name
+
+  policy = jsonencode(
```

Comment:
Just FYI if you want to make this file a bit smaller you could load the policy from a file. Something like that:
```
policy = file("iam_policies/my-policy-file.json")
```

Reply:
created A todo story for this on backlog.  https://github.com/stellar/java-stellar-anchor-sdk/issues/402